server/statistics/store.go

```diff
@@ -184,3 +189,216 @@ func (r *RollingStoreStats) GetKeysReadRate() float64 {
 	defer r.RUnlock()
 	return r.keysReadRate.Median()
 }
+
+const (
+	unknown   = "unknown"
+	labelType = "label"
+)
+
+// ScheduleOptions is an interface to access configurations.
+type ScheduleOptions interface {
+	GetLocationLabels() []string
+	GetMaxStoreDownTime() time.Duration
+	GetLowSpaceRatio() float64
+	GetHighSpaceRatio() float64
+	GetTolerantSizeRatio() float64
+	GetLeaderScheduleLimit(name string) uint64
+	GetRegionScheduleLimit(name string) uint64
+	GetReplicaScheduleLimit(name string) uint64
+	GetMergeScheduleLimit(name string) uint64
+	GetMaxReplicas(name string) int
+	IsRaftLearnerEnabled() bool
+	IsMakeUpReplicaEnabled() bool
+	IsRemoveExtraReplicaEnabled() bool
+	IsRemoveDownReplicaEnabled() bool
+	IsReplaceOfflineReplicaEnabled() bool
+}
+
+type storeStatistics struct {
```

Comment:
Better to a single file? It for the metrics.

Reply:
Yes. But I don't have any good idea to name this file. Any suggestions?